backend/app/serializers/api/v1/concerns.rb

```diff
@@ -0,0 +1,6 @@
+module Api
+  module V1
+    module Concerns
+    end
+  end
+end
```

Comment:
Needed to autoload (zeitwerk) `api/v1/concerns`. Concerns::Notificatable wasn't working in comment/post serializer, for example.